Installation/changes.html

```diff
@@ -269,6 +269,13 @@ <h2 id="release4.8">Release 4.8 </h2>
       <code>CGAL::write_ply_points()</code>
       and <code>CGAL::write_ply_points_and_normals()</code>.</li>
   </ul>
+  <h3>Point Set Shape Detection</h3>
+  <ul>
+    <li>New post-processing
+      algorithm: <code>CGAL::regularize_planes()</code>. This allows the user
+      to favor parallelism, orthogonality, coplanarity and/or axial
+      symmetry between detected planes.</li>
+  </ul>
```

Comment:
The PR is merged in `master` (4.9) but the modification to the `changes.html` is in the section about CGAL-4.8!
@sloriot @sgiraudot